src/types/importCandidate.types.ts

```diff
@@ -25,6 +26,16 @@ export interface ImportCandidate extends Omit<Registration, 'type'> {
   importStatus: ImportStatus;
 }
 
+interface ImportCandidateStatusBucket {
+  key: ImportCandidateStatus;
+  docCount: number;
+}
+export interface ImportCandidateSearchResponse extends Omit<SearchResponse<ImportCandidateSummary>, 'aggregations'> {
```

Comment:
Kan SearchResponse støtte to generics i stedet? 1 for hits og en for aggregations